src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -126,4 +132,11 @@ class EthService(blockchain: Blockchain, appStateStorage: AppStateStorage) {
       highestBlock = appStateStorage.getEstimatedHighestBlock()))
   }
 
+  def sendRawTransaction(req: SendRawTransactionRequest): Future[SendRawTransactionResponse] = {
+    import SignedTransactions._
+    val signedTransaction = rlp.decode[SignedTransaction](req.data.toArray[Byte])
```

Comment:
Should we consider the case when the decoding fails due to a wrongly encoded tx? It seems Geth returns an empty transaction hash if that were to happen.